src/main/java/frc/robot/subsystems/Extension.java

```diff
@@ -122,7 +122,7 @@ public boolean isLimitSwitchPressed() {
      * @param setpoint the setpoint for the subsystem
      */
     public void setSetpoint(ExtensionSetpoints setpoint) {
-        setSetpoint(setpoint.position);
+        setSetpoint(setpoint.position + Dashboard.getInstance().getExtensionOffset());
```

Comment:
Instead of calling dashboard here, I think there should be a setOffset function on the subsystem and the dashboard calls that function in its periodic. Otherwise theres a chance of circular dependency where extension needs dashboard needs extension